app/views/partials/theme/simple/posts/card.liquid

```diff
@@ -26,6 +32,11 @@
                </div> -->
         </div>
         <span>
+          {% if root_post.creator.id == current_profile.id %}
+            <a href="/dashboard/posts/edit?id={{ post.id }}" class="pr-5">
+              {{ 'app.models.posts.edit' | t }}
+            </a>
+          {% endif %}
```

Comment:
This will hide the button, but don't we also need to prevent users from getting to /dashboard/posts/edit manually with the ID that does belong to another user?